### ISSUE TITLE
Version 3.1.1 not available on pypi.org

### ISSUE THREAD

**Issue by dvzrv** [jaraco/jaraco.logging#2]
Hi! I'm currently working on improving the packaging for this project on Arch Linux.
I noticed that version 3.1.1 is tagged in this repository, but it is not available on pypi.org ([current release history](https://pypi.org/project/jaraco.logging/#history) has 3.1.0 from 2022 as the latest release).

As 3.1.1 [deprecates the `jaraco/__init__.py` file](https://github.com/jaraco/jaraco.logging/commit/345978459a345347854b066fe4be01f7a29d03da) this is a rather important change it seems.

Somewhat related to the above mentioned change: Can downstreams use your namespaced packages, if none of them provides the `jaraco/__init__.py` file? If I understand [PEP420](https://peps.python.org/pep-0420/) correctly, an empty file is not required but the [discussion section mentions performance penalties with namespaced packages](https://peps.python.org/pep-0420/#discussion).

**Comment by dvzrv** [comment]
@jaraco ping

**Comment by jaraco** [comment]
> Can downstreams use your namespaced packages, if none of them provides the `jaraco/__init__.py` file? If I understand [PEP420](https://peps.python.org/pep-0420/) correctly, an empty file is not required but the [discussion section mentions performance penalties with namespaced packages](https://peps.python.org/pep-0420/#discussion).

PEP 420 packages _cannot_ contain an `__init__.py` module. If they do, they're now regular packages and won't behave as intended. In particular, if the packages in that namespace exist in more than one path on `sys.path`, only the first one will be recognized.

For example:

```
draft $ mkdir -p ns1/foo
 draft $ touch ns1/foo/bar.py
 draft $ mkdir -p ns2/foo
 draft $ touch ns2/foo/baz.py
 draft $ env PYTHONPATH=ns1:ns2 py -c "import foo.bar; import foo.baz"
 draft $ touch ns1/foo/__init__.py
 draft $ touch ns2/foo/__init__.py
 draft $ env PYTHONPATH=ns1:ns2 py -c "import foo.bar; import foo.baz"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'foo.baz'
 draft $ env PYTHONPATH=ns2:ns1 py -c "import foo.bar; import foo.baz"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'foo.bar'
```

So what that means is that all implementations of packages that use the `jaraco` namespace should _not_ include an `__init__.py`.

**Comment by jaraco** [comment]
Thanks for the ping. I've published v3.1.1 (as originally tagged) and also publishing v3.1.2.
